internal/events/websockets/websockets.go

```diff
@@ -152,3 +152,28 @@ func (ws *WebSockets) WaitClosed() {
 		ws.waitClose()
 	}
 }
+
+func (ws *WebSockets) GetStatus() *fftypes.WebSocketStatus {
+	status := &fftypes.WebSocketStatus{
+		Enabled:     true,
+		Connections: make([]*fftypes.WSConnectionStatus, 0),
+	}
+	for _, c := range ws.connections {
```

Comment:
Do we need to lock here to avoid a read while another goroutine is writing?

Reply:
Yea, good call. Added that.